src/Orleans.Reminders/ReminderService/GrainBasedReminderTable.cs

```diff
@@ -142,8 +143,8 @@ public Task<bool> RemoveRow(GrainReference grainRef, string reminderName, string
                 return Task.FromResult(true);
             }
 
-            logger.LogWarning(
-                (int)ErrorCode.RS_Table_Remove,
+            logger.Warn(
```

Comment:
This is wrong: these helper methods only existed as a temporary solution to migrate away from the old in-build logging system (and to Microsoft.Extensions.Logging.Abstractions), but still stick around until we can finish cleaning them up (we were waiting for analyzers to make sure we don't make mistakes when converting to structured logging)